kong/client.go

```diff
@@ -280,18 +309,20 @@ func (c *Client) Status(ctx context.Context) (*Status, error) {
 	return &s, nil
 }
 
-// Root returns the response of GET request on root of
-// Admin API (GET /).
+// Root returns the response of GET request on root of Admin API (GET / or /kong with a workspace).
 func (c *Client) Root(ctx context.Context) (map[string]interface{}, error) {
-	req, err := c.NewRequest("GET", "/", nil, nil)
+	endpoint := "/"
+	if c.hasWorkspace() {
+		endpoint = "/kong"
+	}
+	req, err := c.NewRequest("GET", endpoint, nil, nil)
 	if err != nil {
 		return nil, err
 	}
-
-	var root map[string]interface{}
-	_, err = c.Do(ctx, req, &root)
+	var informations map[string]interface{}
```

Comment:
Call this `info`:

https://jakubmarian.com/information-is-vs-informations-are-is-information-plural-or-singular-in-english/

Reply:
Betrayed by my native language. Thank you for tip !